Tools/toggle_bitcoin_taproot_bip8_lottrue..sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+if [[ "$0" = "$BASH_SOURCE" ]]; then
+    echo "This script must be sourced \". toggle_bitcoin_taproot_bip8_lottrue.sh\"" 
+    exit 1
+fi
+
+set -e
```

Comment:
since this is being sourced, would it make more sense to restore the previous value or to require it to have already been set (e.g. `set -o | grep errexit || exit 1`)